### PR TITLE
React to SignalR changes

### PR DESCRIPTION
**Pull request by pranavkm**
cc @muratg \ @anurse

### CODE REVIEW

**Comment by muratg** [comment]
cc @anurse

File: packages/packages.csv
Comment (JunTaoLuo) [inline packages/packages.csv:174]: Why is this package listed twice?

Comment (JunTaoLuo) [reply packages/packages.csv:174]: nvm I misread

Comment (davidfowl) [reply packages/packages.csv:174]: Noship?

Comment (pranavkm) [reply packages/packages.csv:174]: It's a package with `.Internal`. We really shouldn't be publishing it to nuget.org

Comment (muratg) [reply packages/packages.csv:174]: We have another .Internal package that ships I believe.  I'd like @anurse to comment because he knows the latest on this.

Comment (analogrelay) [reply packages/packages.csv:174]: It needs to ship along with SignalR, so it needs to be 'ship'. It's `.Internal` to indicate that it is not currently supported on it's own. It's only "supported" via using SignalR.

Comment (davidfowl) [reply packages/packages.csv:174]: Thats what I thought

Comment (pranavkm) [reply packages/packages.csv:174]: By ship, we mean publish to nuget.org when 1.2.0 comes out. Does that sound right? If so, can we come up with a better name for this or make it a `.Sources` package?

Comment (analogrelay) [reply packages/packages.csv:174]: Yeah, publish at 1.2.0. The big reason for the name is that we are already shipping a Microsoft.Extensions.WebSockets, so we'll have to merge the two when we actually come to ship it. I guess we could make it a sources package, or just pull the source in and not ship the package if @davidfowl 's cool with that.

Comment (pranavkm) [reply packages/packages.csv:174]: Filed https://github.com/aspnet/SignalR/issues/24 to track a better Id for those packages and \ or an alternate solution. I'll leave them as noship for now

**Comment by pranavkm** [comment]
bump. Need to get this in to unblock dev

File: packages/packages.csv
Comment (davidfowl) [inline packages/packages.csv]: Should be changed to ship

File: packages/packages.csv
Comment (davidfowl) [inline packages/packages.csv]: Also ship

**Comment by pranavkm** [comment]
🆙 :date

File: packages/packages.csv
Comment (davidfowl) [inline packages/packages.csv:93]: Ship

Comment (pranavkm) [reply packages/packages.csv:93]: I thought we were planning on renaming these first: https://github.com/aspnet/SignalR/issues/24

Comment (davidfowl) [reply packages/packages.csv:93]: Does this affect anything right now? If not that's fine.

**Review by davidfowl — APPROVED** [review]

File: packages/packages.csv
Comment (pranavkm) [reply packages/packages.csv:93]: Nope. Just makes sure we don't ignore the work item :)

**Pull request merged**
